src/api/exporters/ezsLocals.js

```diff
@@ -84,7 +85,26 @@ export function JSONLDObject(data, feed) {
                 }
             }
         });
-        feed.send(output);
+
+        const transformCompleteFields = async (field) => {
```

Comment:
The function does not return anything. If you mutate, use `forEach` instead of map but you should avoid mutating parameters. I would keep a list of field without `completes` property and create a new array of fields with `completes` property, using the map function to get a transformed version of them. I would then combine the two lists.